scripts/hash-exec-copy.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+URL=$(echo "$1" | cut -f2 -d=)
```

Comment:
I think it would be better to just take the variable as is, and remove the 2nd option of taking anything that comes after `=`. As it is now, you can run this script with `nonesense=<url>` and it would still work.

So in general we should either parse an exact field name, or not support giving the field name at all. For the case where there is only one input variable I see we mostly use the second option in the existing scripts (which is simpler).

Reply:
see for example `archive-dssspell.sh`.

Reply:
yes, I agree, for one arg is better the simple pattern, don in 748dcb